tests/server.py

```diff
@@ -307,16 +308,28 @@ def connectTCP(self, host, port, factory, timeout=30, bindAddress=None):
         return conn
 
     def advance(self, amount):
-        # run any "callFromThread" callbacks before anything registered with
-        # "callLater"
+        # first advance our reactor's time, and run any "callLater" callbacks that
+        # makes ready
```

Comment:
This is indeed easier to understand - though I'm not sure if "makes ready" is a typo.

Reply:
it's not a typo, though there may be better ways of expressing it.

Suppose our fake reactor thinks it is 12:00:00, and we have registered a `callLater` callback for 12:00:02. `advance(5)` means that the callback will now be "ready" to run: it was "made ready" by advancing the reactor.